aries-mobile-tests/features/steps/bc_wallet/offline_handling.py

```diff
@@ -23,10 +23,10 @@
 def step_impl(context):
     if context.driver.capabilities['platformName'] == 'iOS':
         height = WebDriverWait(context.driver, 30).until(
-                EC.presence_of_element_located((MobileBy.CLASS_NAME, "UIAWindow"))
+                EC.presence_of_element_located((AppiumBy.CLASS_NAME, "UIAWindow"))
             ).size["height"]
         width = WebDriverWait(context.driver, 30).until(
-                EC.presence_of_element_located((MobileBy.CLASS_NAME, "UIAWindow"))
+                EC.presence_of_element_located((AppiumBy.CLASS_NAME, "UIAWindow"))
             ).size["width"]
         if '6' in context.driver.capabilities['testobject_device_name']:
```

Comment:
Not in this PR but 6 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)). When you get the change, consider defining them as `has_face_id_capabilitie = 6` then use the var name in code.